sdk/tests/conformance2/transform_feedback/too-small-buffers.html

```diff
@@ -0,0 +1,242 @@
+}
+</script>
+<script id="fshader" type="x-shader/x-fragment">#version 300 es
+precision mediump float;
+out vec4 dummy;
+void main() {
+  dummy = vec4(0.);
+}
+</script>
+<script>
+"use strict";
+description("Transform feedback into buffers that are too small should produce errors.");
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+var gl = wtu.create3DContext(canvas, null, 2);
+
+if (!gl) {
+    testFailed("WebGL context does not exist");
+} else {
+    testPassed("WebGL context exists");
+}
+
+const progInterleaved = wtu.setupTransformFeedbackProgram(gl, ["vshader", "fshader"],
+    ["out_value1", "out_value2"], gl.INTERLEAVED_ATTRIBS,
+    ["in_value1", "in_value2"]);
+const progSeparate = wtu.setupTransformFeedbackProgram(gl, ["vshader", "fshader"],
+    ["out_value1", "out_value2"], gl.SEPARATE_ATTRIBS,
+    ["in_value1", "in_value2"]);
+wtu.glErrorShouldBe(gl, gl.NO_ERROR, "program compilation");
+
+// Attrib 1 contains 4 vertices. Attrib 2 contains 4 instance indices.
+const vertexBuffer0 = gl.createBuffer();
+gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer0);
+gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([1, 2, 3, 4]), gl.STATIC_DRAW);
+const vertexBuffer1 = gl.createBuffer();
+gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer1);
+gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([1, 2, 3, 4]), gl.STATIC_DRAW);
+gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer0);
+gl.enableVertexAttribArray(0);
+gl.vertexAttribPointer(0, 1, gl.FLOAT, false, 0, 0);
+gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer1);
+gl.enableVertexAttribArray(1);
+gl.vertexAttribPointer(1, 1, gl.FLOAT, false, 0, 0);
+gl.vertexAttribDivisor(1, 1);
+
+let tfBuffer0 = gl.createBuffer();
+let tfBuffer1 = gl.createBuffer();
+
+wtu.glErrorShouldBe(gl, gl.NO_ERROR, "setup");
+
+let cases = [
+    { name: "drawArrays",
+      drawFunction: ()=>gl.drawArrays(gl.POINTS, 0, 4),
+      result: [[2, 4, 6, 8], [2, 2, 2, 2]]},
+    { name: "drawArraysInstanced one instance",
+      drawFunction: ()=>gl.drawArraysInstanced(gl.POINTS, 0, 4, 1),
+      result: [[2, 4, 6, 8], [2, 2, 2, 2]]},
+    { name: "drawArraysInstanced four instances",
+      drawFunction: ()=>gl.drawArraysInstanced(gl.POINTS, 0, 1, 4),
+      result: [[2, 2, 2, 2], [2, 4, 6, 8]]},
+  ];
+
+for (let {name, drawFunction, result} of cases) {
+  debug("<h1>" + name + "</h1>")
+
+  let interleavedResult = [];
+  for (let i = 0; i < result[0].length; i++) {
+    interleavedResult.push(result[0][i], result[1][i]);
+  }
+
+  let doTransformFeedback = (drawFunction, error) => {
+    gl.beginTransformFeedback(gl.POINTS);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "before draw");
+    drawFunction();
+    wtu.glErrorShouldBe(gl, error, "draw");
+    gl.endTransformFeedback();
+  }
+
+  gl.useProgram(progInterleaved);
+
+  debug("<h3>interleaved - Baseline success case</h3>")
+  gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, tfBuffer0);
+  gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 8*4, gl.STREAM_READ);
```

Comment:
nit: if you use a const sizeOfFloat instead of 4, it's easier to read. Here and below.

Reply:
Done.